torch/autograd/graph.py

```diff
@@ -133,6 +134,14 @@ def __subclasshook__(cls, C):
         return NotImplemented
 
 
+GradientEdge = namedtuple("GradientEdge", ("node output_nr"))
+GradientEdge.__doc__ = """\
+Object representing a given gradient edge within the autograd graph.
+To get the gradient edge where a given Tensor gradient will be computed,
+you can do ``edge = GradientEdge(t.grad_fn, t.output_nr)``.
+"""
```

Comment:
Do we want to mention how its used here? Like computing gradient wrt `t` is equivalent to computing gradient wrt to its gradient edge.

Reply:
It would seem convenient to have a helper API that allows users to create a GradientEdge from a tensor `edge = get_gradient_edge(x)`

Reply:
And we can do the fancy trick to get the accumulate grad node in there? I like that